package.json

```diff
@@ -66,23 +76,26 @@
     "eventsource-polyfill": "^0.9.6",
     "express": "^4.14.0",
     "extract-text-webpack-plugin": "^2.1.0",
-    "file-loader": "^0.9.0",
-    "html-webpack-plugin": "^2.24.1",
-    "image-webpack-loader": "^3.0.0",
+    "file-loader": "^3.0.1",
+    "html-webpack-plugin": "^3.2.0",
+    "image-webpack-loader": "~4.6.0",
     "json-loader": "^0.5.4",
-    "mocha": "^3.2.0",
+    "mini-css-extract-plugin": "^0.5.0",
+    "mocha": "~4.0.0",
     "nib": "^1.1.2",
     "publisssh": "^1.1.0",
     "react-addons-test-utils": "^15.4.2",
+    "react-hot-loader": "~3.0.0-beta.7",
```

Comment:
I believe hot module functionality is built into webpack-dev-server, so I don't think you need this anymore: https://webpack.js.org/configuration/dev-server/#devserverhot